src/i18n/locales/de/default.json

```diff
@@ -155,7 +155,8 @@
     "vocabs": "Vokabeln",
     "vocabsPerDay": "Vokabeln pro Tag",
     "vocascan": "Vocascan",
-    "wrong": "Falsch"
+    "wrong": "Falsch",
+    "search": "Suchen"
```

Comment:
remove these changes

Reply:
Already done in https://github.com/vocascan/vocascan-frontend/pull/110

Reply:
Yeah, but you also need to remove them here. If we merge that, this change is also included.

Reply:
If we merge https://github.com/vocascan/vocascan-frontend/pull/110, its gone

Reply:
Thats how git works

Reply:
Have cherry-picked the commit. Its gone now